### PR TITLE
Bbs release v4.1.de.md

### PR DESCRIPTION
**Pull request by Surf-N-Code**
Initial translation to German

### CODE REVIEW

File: content/bbs/bbs-release-v4.1.de.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
++++
+
+Skycoin BBS v4.1 wurde released!
+
+[![Skycoin BBS Showcase 4 - YouTube](https://i.ytimg.com/vi/6ZqwgefYauU/0.jpg)](https://youtu.be/6ZqwgefYauU)
+
+## Änderungen
+- Verbesserung der Benutzeroberfläche.
+- Implementierung der "service discovery" (Verbindungen zu anderen Nodes werden nun automatisch verwaltet).
+- Verbesserung des Einreichens von Foreninhalten (Implementiert durch "service discovery").
```
Comment (IamVanCleef) [inline content/bbs/bbs-release-v4.1.de.md]: implementiert

File: content/bbs/bbs-release-v4.1.de.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Vereinfachter Prozess zur Erstellung von Foren und deren Hosting.
+- Verbesserung der CXO Datenstruktur.
```
Comment (IamVanCleef) [inline content/bbs/bbs-release-v4.1.de.md]: CXO-Datenstruktur

File: content/bbs/bbs-release-v4.1.de.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Das nächste Release wird viele weitere spannende Features bringen! Bleibe auf dem laufenden und schaue hier erneut vorbei!
```
Comment (IamVanCleef) [inline content/bbs/bbs-release-v4.1.de.md]: auf dem Laufenden

**Comment by IamVanCleef** [comment]
Is anyone working at this? If yes, it would be nice if the changes could be done so that the file can be merged. if not, then I am looking forward to translate some stuff...

**Comment by C3POv7** [comment]
Do you need any help with this one?

**Comment by Surf-N-Code** [comment]
Done

**Pull request merged**